core/src/main/java/com/graphhopper/routing/util/CustomArea.java

```diff
@@ -35,7 +34,7 @@ public static CustomArea fromJsonFeature(JsonFeature j) {
         for (int i = 0; i < j.getGeometry().getNumGeometries(); i++) {
             Geometry geometry = j.getGeometry().getGeometryN(i);
             if (geometry instanceof Polygon) {
-                PolygonExtracter.getPolygons(geometry, borders);
+                borders.add((Polygon) geometry);
```

Comment:
Just out of curiosity, can you explain why this changed?

Reply:
Ah yeah. For some reason I thought this was related to the no longer needed GeometryFactory. And yes, definitely more readable :)